assets/first_party/knitwitch/Blue/Knitwitch_Appear.png.import

```diff
@@ -0,0 +1,34 @@
+[remap]
+
+importer="texture"
+type="CompressedTexture2D"
+uid="uid://cs0aa2jajjnil"
+path="res://.godot/imported/Knitwitch_Appear.png-0b673a0101d49780a949012c7bf58904.ctex"
+metadata={
+"vram_texture": false
+}
+
+[deps]
+
+source_file="res://assets/first_party/knitwitch/Blue/Knitwitch_Appear.png"
```

Comment:
I put these in `assets` but for most other props we've placed the assets directly into the relevant point of the `scenes` hierarchy. WDYT?

Reply:
Yes, we are using 2 folder structures. I would like to see the files in `assets/first_party/` distributed in the `components/` subfolders of game elements. But is true that if the same asset is used in multiple game elements, it becomes tricky. What do you think?